tests/manage/monitoring/conftest.py

```diff
@@ -838,41 +841,61 @@ def delete_target_bucket():
 
 
 @pytest.fixture
-def measure_stop_worker_node(measurement_dir, nodes):
+def measure_stop_worker_nodes(request, measurement_dir, nodes):
     """
-    Stop one worker node, measure the time when it was stopped and monitors
-    alerts that were triggered during this event.
+    Stop worker nodes that doesn't contain RGW (so that alerts are triggered
+    correctly), measure the time when it was stopped and monitors alerts that
+    were triggered during this event.
 
     Returns:
         dict: Contains information about `start` and `stop` time for stopping
             worker node
 
     """
-    node = get_nodes(node_type="worker")[0]
-
-    def stop_node():
+    mgr_pod = pod.get_mgr_pods()[0]
+    mgr_node = pod.get_pod_node(mgr_pod)
+    test_nodes = [
+        worker_node
+        for worker_node in get_nodes(node_type="worker")
```

Comment:
Not mandatory, but we can change to `get_nodes(node_type=constants.WORKER_MACHINE)`.